manifests/t/the-sz/Banks/1.02/the-sz.Banks.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: the-sz.Banks
+PackageVersion: 1.02
+InstallerType: zip
+Installers:
+- Architecture: x86
+  InstallerUrl: https://the-sz.com/common/get.php?product=banks

```

Comment:
Missing NestedInstallerType and NestedInstallerFiles

Reply:
NestedInstallerType and NestedInstallerFiles added